modules/tutorials/pages/buckets-scopes-and-collections.adoc

```diff
@@ -0,0 +1,108 @@
+= Buckets, Scopes and Collections
+:description: In this section, you'll learn how to logically partition your data in Couchbase using buckets, scopes and collections.
+:page-pagination: full
+
+include::partial$tutorial-globals.adoc[]
+
+[abstract]
+{description}
+
+== So What is a Bucket Exactly?
+
+If you think in database terms, a Couchbase Bucket is analogous to a database: it's the data store where you're going to store and retrieve related information about the students.
+
+You can click on the menu:Dashboard[Buckets] link to access the Buckets page, then click on btn:[Add Bucket].
+
+In this dialog, enter `student-bucket` in the *Name* box.
+
+image::add-student-bucket.png[alt="Adding student bucket to Couchbase"]
+
+Once you have entered the bucket name, press the btn:[Add Bucket] button to return to the main bucket list.
+
+
+== Scopes and Collections
+
+In all but the simplest cases, it's better to provide some kind of separation between documents of different types.
+Couchbase has a simple hierarchy model which allows for such separation:
+
+image::multi-level-data-containment.png[alt="Couchbase data hierarchy"]
```

Comment:
Just noticed that this image has a disclaimer at the bottom left saying "Confidential and proprietary: do not distribute without Couchbase consent"! It's not a problem for now, but I think we should eventually replace this image, e.g. with something we have generated in UML.

Reply:
Good point.  I'll put something together for the follow up release.